package.json

```diff
@@ -3,7 +3,10 @@
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
     "build": "eslint src && pulp build -- --censor-lib --strict",
-    "test": "pulp test"
+    "test": "pulp test",
+    "bench:build": "purs compile \"bench/**/*.purs\" \"src/**/*.purs\" \"bower_components/*/src/**/*.purs\"",
```

Comment:
Sorry I didn't spot this before - I would prefer `pulp build --include bench` for this, if I've got that right.